### PR TITLE
tome4: 1.4.6 -> 1.4.9

### PR DESCRIPTION
**Pull request by rardiol**
###### Motivation for this change

update and fix compile failure

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

maintainer @Chattered

### CODE REVIEW

**Comment by Chattered** [comment]
##### Reviewed points

- [X] package name fits guidelines
- [X] package version fits guidelines
- [X] package build on x86-64/NixOS
- [X ] executables tested on x86-64/NixOS

File: pkgs/games/tome4/default.nix [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
   NIX_CFLAGS_COMPILE = [ "-I${SDL2_image}/include/SDL2" "-I${SDL2_ttf}/include/SDL2" ];
   preConfigure = ''
-    sed -e "s@/opt/SDL-2.0/include/SDL2@${SDL2}/include/SDL2@g" -e "s@/usr/include/GL@/run/opengl-driver/include@g" -i premake4.lua
+    sed -e "s@/opt/SDL-2.0/include/SDL2@${SDL2.dev}/include/SDL2@g" -e "s@/usr/include/GL@/run/opengl-driver/include@g" -i premake4.lua
     premake4 gmake
   '';
   makeFlags = [ "config=release" ];
   installPhase = ''
     install -Dm755 t-engine $out/opt/tome4/t-engine
+    echo "#!/{bash}/bin/sh" >> tome4
```
Comment (joachifm) [inline pkgs/games/tome4/default.nix]: You can also do `${stdenv.shell}` here, or simply use `/bin/sh` and rely on shebang patching to fixup the path.

Comment (FRidh) [inline pkgs/games/tome4/default.nix]: in any case it is missing the $

Comment (rardiol) [inline pkgs/games/tome4/default.nix]: Ok, changed to /bin/sh

**Comment by Mic92** [comment]
Merged with some cosmetical changes in 8cbba14dbe34dd2d7f838e4322976c8350f02b6f

**Pull request closed without merging**